core/src/i18n/en.yaml

```diff
@@ -716,12 +716,25 @@ functions:
       text: Warning
       variable-can-override-sbu: This variable can override or be overridden by a variable injected by one of the created ServiceBindings.
 gateways:
+  name_singular: Gateway
+  description: Gateway describes a load balancer operating at the edge of the mesh receiving incoming or outgoing HTTP/TCP connections.
   create-modal:
+    at-least-one-server-required: Gateway must have at least one server.
+    tooltips:
+      ca-certificates: The path to a file containing certificate authority certificates to use in verifying a presented client side certificate.
+      selector: One or more labels that indicate a set of pods on which the gateway configuration should be applied. By default workloads are searched across all namespaces based on label selectors.
+      protocol: The protocol exposed on the port. TLS implies the connection will be routed based on the SNI header to the destination without terminating the TLS connection.
+      target-port: The port number on the endpoint where the traffic will be received.
+      private-key: The path to the file holding the server’s private key.
+      server-certificate: The path to the file holding the server-side TLS certificate to use.
+      credential-name: 'The name of the Secret that holds the TLS certs including the CA certificates. The secret (of type generic) should contain the following keys and values: key: <privateKey> and cert: <serverCert>. For mutual TLS, cacert: <CACertificate> can be provided in the same secret or a separate secret named <secret>-cacert. Secret of type tls for server certificates along with ca.crt key for CA certificates is also supported.'
```

Comment:
```suggestion
      credential-name: 'The name of the Secret that holds the TLS certs including the CA certificates. The secret (of type generic) must contain the following keys and values: key: <privateKey> and cert: <serverCert>. For mutual TLS (mTLS), a CA certificate (cacert): <CACertificate> can be provided in the same secret or in a separate secret named <secret>-cacert. Secret of type tls for server certificates along with ca.crt key for CA certificates is also supported.'
```
Why does this string start and end with ` ` `, but none of the others?

I assume it's a hard requirement (=> must...), not just a recommendation (should...)? If it's a recommendation, I'd suggest to write "We recommend that the secret (...) contains (...)" instead of "should".

Still trying to understand which kinds of credential name / certificate types are available.

Reply:
This is because it contains ':' character - YAML parser gets confused, so we wrap entire string with `'` to escape the ':'.

Reply:
This is a requirement.
About the change
```
For mutual TLS, cacert: <CACertificate> 
->
For mutual TLS (mTLS), a CA certificate (cacert): <CACertificate> 
```
We cannot write it like this, because it continues the pattern from previous sentence - they enumerated key and values, so keys: (key, cert, cacert) and values: (<privateKey>, <serverCert> and <CACertificate>).

Damn, they did made their docs confusing...
![image](https://user-images.githubusercontent.com/10504661/134926288-49e50213-1de7-4c26-8fce-a0792ef33389.png)

Reply:
:D It surely made total sense to everyone who is already familiar with that stuff!

Reply:
> This is a requirement.
> About the change
> 
> ```
> For mutual TLS, cacert: <CACertificate> 
> ->
> For mutual TLS (mTLS), a CA certificate (cacert): <CACertificate> 
> ```
> 
> We cannot write it like this, because it continues the pattern from previous sentence - they enumerated key and values, so keys: (key, cert, cacert) and values: (, and ).
> 
> Damn, they did made their docs confusing...
> ![image](https://user-images.githubusercontent.com/10504661/134926288-49e50213-1de7-4c26-8fce-a0792ef33389.png)

arrgh I'm too confused to improve it though :(

Reply:
Oh, great, only now I noticed that I can't put words between '<' and '>', because they disappear. Oh well...

Reply:
Maybe just reword it to make it simpler?
```
The name of the Secret. Required properties are key and cert, in case of mutual TLS (mTLS) caceert property is also required. Secret of type tls for server certificates along with ca.crt key for CA certificates is also supported.'
```
I'd go for less information - just give clues, and if users need more info they can check out the other sources, not only istio docs.

Reply:
I agree. They just need a starting point to know what to look for.